mithril-test-lab/mithril-end-to-end/src/utils/file_utils.rs

```diff
@@ -22,6 +22,47 @@ pub async fn tail(file_path: &Path, number_of_line: u64) -> StdResult<String> {
     String::from_utf8(tail_result.stdout).with_context(|| "Failed to parse tail output to utf8")
 }
 
+/// Return the lastest errors in a file with the lines preceding them.
+///
+/// For the sake of simplicity it use internally the `tail` and `grep` commands so be sure to have them on
+/// your system.
+pub async fn last_errors(file_path: &Path, number_of_error: u64) -> StdResult<String> {
```

Comment:
Can we fallback to previous version of logs dump if `grep` or `tail` is missing?